components/ipfs/browser/ipfs_not_connected_page.h

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_IPFS_BROWSER_IPFS_NOT_CONNECTED_PAGE_H_
+#define BRAVE_COMPONENTS_IPFS_BROWSER_IPFS_NOT_CONNECTED_PAGE_H_
+
+#include <memory>
+#include <string>
+
+#include "base/values.h"
+#include "components/security_interstitials/content/security_interstitial_page.h"
+#include "url/gurl.h"
+
+namespace content {
+class WebContents;
+}  // namespace content
+
+namespace security_interstitials {
+class SecurityInterstitialControllerClient;
+}  // namespace security_interstitials
+
+namespace ipfs {
+
+class IPFSNotConnectedPage
```

Comment:
can you please briefly describe what this page is for?

Reply:
Fixed in https://github.com/brave/brave-core/pull/6782/commits/cb284f67a1b05ae52add31a6f437a8a2a7eb384f